sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/TraceHelper.cs

```diff
@@ -49,7 +49,7 @@ internal static List<TelemetryItem> OtelToAzureMonitorTrace(Batch<Activity> batc
                             {
                                 BaseType = "RequestData",
                                 BaseData = activityTagsProcessor.activityType.HasFlag(OperationType.V2)
-                                                ? new RequestData(Version, activity, ref activityTagsProcessor, schemaVersion: "V2")
+                                                ? new RequestData(Version, activity, ref activityTagsProcessor, schemaVersion: "New")
```

Comment:
nit: instead of using "New", let's define a const `DefaultSchemaVersion = 1.0.0` and use that.

Reply:
Fixed.